frontend/apps/reader/modules/readerview.lua

```diff
@@ -226,7 +228,7 @@ function ReaderView:paintTo(bb, x, y)
         -- FIXME: Make sure this doesn't blow up on engines where getDrawnImagesStatistics is not implemented!
         local img_count, img_coverage = self.ui.document:getDrawnImagesStatistics()
         if img_count > 0 and img_coverage >= 0.10 then
```

Comment:
You'll get nil for both on other engines, so yes, it's blow up :) unless `if img_count and img_count > 0 ...`
I guess you can go on with that now that it's merged.